enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -37,6 +38,18 @@ export const PublicComponentStylesWrapper = styled.div`
     display: inline;
   }
 `;
+
+export const PublicComponentStylesWrapper = (
```

Comment:
Had to make a wrapper (on the wrapper 🫠) to add the `mb-wrapper` className on all instances. The class is needed to make the component work correctly.